src/app/appActions.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import type { Action, Dimensions, Dispatch, GetState } from '../types';
```

Comment:
Why did you rename this type?

Reply:
I want to import `Dimensions` from `react-native` (https://github.com/jainkuniya/zulip-mobile/blob/50aa4611fb75e6aece3801137c94b5e85a5fad03/src/compose/ComposeBox.js#L3) but `Dimensions` was already imported from `types` (https://github.com/zulip/zulip-mobile/blob/master/src/compose/ComposeBox.js#L13)

Reply:
Then, rename this to `DimensionsType`?
`SafeAreaDimensions` is just not correct.

Reply:
That being said, I don't think the `Dimensions.get()` is even the correct approach.
Just setting an `absolute` position for the popup and aligning it to the bottom is likely going to work better.